### DISCUSSION TITLE
美国原神

### DISCUSSION THREAD

**Discussion by cpu644** [share121/inter-knot#1583] [其他]
你说的对，但是命运2是一款由Bungie主力开发并由动视在PlayStation 4、Xbox One以及Windows平台上发行的动作角色扮演大型多人在线第一人称射击游戏。本作是2014年发售的《命运》的正统续作。游戏相比前作取消了PlayStation 3和Xbox 360平台并增加了Windows平台，以及增加了对繁简中文的支持。
Bungie于2017年3月30日正式发布了《命运2》的宣传视频。游戏于2017年9月6日发售Xbox One及PlayStation 4版本，新增加的Windows平台10月24日在暴雪战网发售；Windows版本由动视的兄弟公司暴雪娱乐负责运营。在Bungie与动视分手后，游戏的Windows版本于2019年登陆Steam网络商店，并且增加了对谷歌的云游戏平台[Google Stadia]的支持。2020年5月，Bungie确认《命运2》将会登陆新世代主机PlayStation 5和Xbox Series X/S。该游戏以玩家对环境（PvE）及玩家对战（PvP）为主。玩家对环境下，有诸如迷宫探索（游戏中分为两种，地牢与“打击”，三人组队）、突袭（六人组队，通常由复杂的地图和游戏机制组成）等游戏模式。
![屏幕截图 2024-08-11 190558](https://github.com/user-attachments/assets/045bf9b8-1e15-417c-965e-15635cc1dd67)

**Comment by molehzy** [comment]
怎么哪里都有噶殿(脑
